common/README.ipynb

```diff
@@ -32,18 +41,26 @@
     "* Choose \"Boot Mode\" tab on left (play button) \n",
     " After changing these settings, the notebook will no longer start as JupyterLab but as Voila. To learn more about Voila, see here: https://github.com/voila-dashboards/voila\n",
     "\n",
+    "#### Octave Kernel\n",
+    "\n",
+    "This kernel can run cells in both octave and python but, please note that **variables are not shared across different kernels**. Nonetheless there are alternatives\n",
+    "to share data between kernels like the magic command [``%store``](https://www.blopig.com/blog/2020/05/storing-variables-in-jupyter-notebooks-using-store-magic/). For more details see [#301](https://github.com/ITISFoundation/osparc-issues/issues/301).\n",
+    "\n",
     "#### Have an issue?\n",
     "\n",
-    "Please open an issue on GitLab at [oSparc/services/jupyter-smash](https://git.speag.com/oSparc/services/jupyter-smash)"
+    "Please open an issue on GitLab at [ITISFoundation/jupyter-math/issues](https://github.com/ITISFoundation/jupyter-math/issues)"
```

Comment:
please use the link to this repo, not math!

Reply:
Done, thanks for spotting this. I forgot to save the file earlier...